package.json

```diff
@@ -1,10 +1,15 @@
 {
-  "name": "ZulipNative",
-  "version": "0.0.1",
+  "name": "ZulipMobile",
+  "version": "0.9.0",
   "private": true,
   "scripts": {
     "start": "react-native start",
-    "ios": "react-native run-ios",
+    "ios-min": "react-native run-ios --simulator 'iPhone 5'",
+    "ios-max": "react-native run-ios --simulator 'iPhone 7 Plus'",
+    "ios-device": "react-native run-ios --device 'Max's iPhone'",
```

Comment:
Because it almost always is called something like 'Boris's iPhone' or 'Neeraj's iPhone'. User might need to edit that, if not called Max ;)
It seems not specifying the name also works, so I'll keep it to '--device' only.